src/java/org/apache/cassandra/auth/CassandraRoleManager.java

```diff
@@ -301,6 +306,7 @@ public void dropRole(AuthenticatedUser performer, RoleResource role) throws Requ
                 consistencyForRoleWrite(role.getRoleName()));
         removeAllMembers(role.getRoleName());
         removeAllIdentitiesOfRole(role.getRoleName());
+        disconnect(role);
```

Comment:
I think I'll remove ALTER-coordinator disconnects here, seems better to have all disconnects happen on the jitter schedule (particularly for smaller clusters)